src/azure-cli/azure/cli/command_modules/cdn/commands.py

```diff
@@ -207,3 +285,117 @@ def load_command_table(self, _):
         g.custom_command('delete', 'delete_waf_rate_limit_rule', client_factory=cf_waf_policy, confirmation=True)
         g.custom_command('list', 'list_waf_rate_limit_rules', client_factory=cf_waf_policy)
         g.custom_show_command('show', 'show_waf_rate_limit_rule', client_factory=cf_waf_policy)
+
+    with self.command_group('afd profile', cdn_profiles_sdk,
+                            is_preview=True,
```

Comment:
>is_preview=True [](start = 28, length = 15)

We can define `is_preview=True` for command group `afd`, then all sub command groups will show as preview.

Reply:
Thanks, fiexed.